.gitignore

```diff
@@ -6,4 +6,5 @@ mwb-connect-firebase-adminsdk.json
 mwb-connect-firebase-adminsdk.prod.json
 \node_modules
 \dist
-\.wwebjs_auth
\ No newline at end of file
+\.wwebjs_auth
+.idea
```

Comment:
otherwise I always have to look out for mistakenly committing those files. I cant delete them form my machine